src/components/careers/ExpandableJob.tsx

```diff
@@ -3,112 +3,166 @@ import React, { useState } from 'react'
-    border-radius: 3px;
-
-    @media(max-width: 600px) {
-        padding: 4rem 2rem 2rem 8rem;
+  position: relative;
+  padding: 4rem 4rem 4rem 10rem;
+  background: ${colors.offWhite};
+  border: ${borders.light2};
+  border-radius: 3px;
+
+  @media (max-width: 600px) {
+    padding: 4rem 2rem 2rem 8rem;
+  }
+
+  button {
+    position: absolute;
+    top: 4rem;
+    left: 4rem;
+    border: none;
+    transform: translateY(-0.6rem);
+
+    @media (max-width: 600px) {
+      left: 2rem;
     }
 
-    button {
-        position: absolute;
-        top: 4rem;
-        left: 4rem;
-        border: none;
-        transform: translateY(-.6rem);
-
-        @media(max-width: 600px) {
-            left: 2rem;
-        }
-
-        &:hover {
-            svg {
-                fill: ${colors.text};
-            }
-        }
+    &:hover {
+      svg {
+        fill: ${colors.text};
+      }
     }
+  }
 
-    p + p {
-        margin-top: 2rem;
-    }
+  p + p {
+    margin-top: 2rem;
+  }
 
-    .after {
-        margin: 5rem 0;
-    }
+  .after {
+    margin: 5rem 0;
+  }
 
-    .btn {
-        margin-bottom: 5rem;
-    }
+  .btn {
+    margin-bottom: 5rem;
+  }
 
-    h4 {
-        margin: 5rem 0 2rem;
-    }
+  h4 {
+    margin: 5rem 0 2rem;
+  }
 
-    ul {
-        list-style: initial;
-        list-style-position: inside;
-    }
+  ul {
+    list-style: initial;
+    list-style-position: inside;
+  }
 `
 
 interface List {
-    title: string
-    items: string[]
+  title: string
+  items: string[]
 }
 
 export interface ExpandableJobProps {
-    title: string
-    intro: string
-    paragraphs: string
-    lists: List[]
-    textAfterTheLists: string
-    rendered?: boolean
+  title: string
+  intro: string
+  paragraphs: string
+  lists: List[]
+  textAfterTheLists: string
+  rendered?: boolean
+  date?: string
 }
 
-const ExpandableJob = ({ title, intro, paragraphs, lists, textAfterTheLists, rendered }: ExpandableJobProps) => {
-    const [isRendered, setIsRendered] = useState<boolean>(rendered || false)
-
-    const toggleIsRendered = () => {
-        setIsRendered(!isRendered)
+const ExpandableJob = ({ title, intro, paragraphs, lists, textAfterTheLists, rendered, date }: ExpandableJobProps) => {
+  const [isRendered, setIsRendered] = useState<boolean>(rendered || false)
+
+  const toggleIsRendered = () => {
+    setIsRendered(!isRendered)
+  }
+
+  const applicationEmail = `mailto:contact@typefox.io?subject=Application as ${title}`
+
+  const structuredData = {
+    '@context': 'https://schema.org/',
+    '@type': 'JobPosting',
+    title: title,
+    description: `<p>${intro}</p>${paragraphs.split('\n').map((p) => `<p>${p}</p>`)}`,
+    datePosted: date,
+    hiringOrganization: {
+      '@type': 'Organization',
+      name: 'TypeFox GmbH',
+      sameAs: 'https://www.typefox.io/',
+      logo: TypeFoxLogo,
+      contactPoint: {
+        '@type': 'ContactPoint',
+        url: 'https://www.typefox.io/contact/'
+      }
+    },
```

Comment:
I think, going forward, we'll want Gitpod's hiring organization to be "Gitpod Inc." instead of "TypeFox GmbH":

```suggestion
    hiringOrganization: {
      '@type': 'Organization',
      name: 'Gitpod Inc.',
      sameAs: 'https://www.gitpod.io/',
      logo: GitpodLogo,
      contactPoint: {
        '@type': 'ContactPoint',
        url: 'https://www.gitpod.io/contact/'
      }
    },
```